core/src/impl/Kokkos_Utilities.hpp

```diff
@@ -65,6 +65,9 @@ struct identity {
 template <typename T>
 using identity_t = typename identity<T>::type;
 
+template <typename... Is>
```

Comment:
Why not a template alias:

```c++
template<typename...>
using always_true = std::true_type;

```
(`std::void_t` does a similar thing.)

Reply:
Good point.  Will change.

Reply:
No I remember I had tried that before it does not compile.
For instance https://github.com/kokkos/kokkos/runs/5385474386?check_suite_focus=true

Reply:
Will push force back the commit that had passed CI when you confirm you saw that